app/controllers/items_controller.rb

```diff
@@ -196,7 +196,7 @@ def set_rights
     render status: :bad_request, plain: 'Invalid new rights setting.'
   end
 
-  # set the rightsMetadata to the AdminPolicies' defaultObjectRights
+  # set the object's access to its' admin policy's defaultAccess
```

Comment:
"its" not "its'"

Reply:
@justinlittman Good eye! Fixed in a follow-up drive-by commit. If you approve, can you squash-merge?